Enabled --safe-updates in bench mariadb

ABHISHEK-KEDAR-21 opened the pull request:
Enabled --safe-updates in bench mariadb for doing safe update operations in the database.

ABHISHEK-KEDAR-21 committed:
Enabled --safe-updates in bench mariadb

AKedar21 commented:
This is only enabled in "bench mariadb" command which i believe is not used inside code to implement fuctionalities which might break. Also you can always put where 1=1 in SQL query (an always true condition), a choice which will explicitly state that whole table must be affected, ensuring a condition miss in not a mistake.

sunnyakaxd commented on frappe/commands/utils.py:
```diff
@@ -342,7 +342,7 @@ def mariadb(context):
 		frappe.conf.db_name,
 		'-h', frappe.conf.db_host or "localhost",
-		'--pager=less -SFX',
+		'--pager=less -SFX','--safe-updates',
```
Add a newline here

AKedar21 replied:
Thanks for the suggestion, added a newline and committed again.

ABHISHEK-KEDAR-21 committed:
Enabled mysql --safe-updates in bench mariadb

This will ensure that --safe-updates is enabled in bench mariadb which will avoid accidental updates in the table if where or limit clause is missing in the SQL query.
